Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+# Gopkg.toml example
```

Comment:
Since we have the vendor directory, it's not really important to keep Gopkg files.

Reply:
@erbesharat I see, but if I am not wrong that gets created while using go dep, so does that mean we have to delete manually every time?

Reply:
In general they are not needed, but they don't hurt either. One could use .gitignore to not commit them or show them as change.